### PR TITLE
Small typo/grammar suggestions

### PR DESCRIPTION
**Pull request by jcarolinares**
## What This PR Changes
-Small typo/grammar suggestions

Hello, @karlsoderby I'm playing a little bit with a Minima and following the tutorial I found some potential/typos and grammar suggestions :)

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/02.hero/boards/uno-r4-minima/tutorials/usb-hid/usb-hid.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ This feature can be used to create game controllers, keyboard extensions or othe
 
 ## Goals
 
-The goals of this tutorials are:
+The goals of this tutorial is:
```
Comment (jhansson-ard) [inline content/hardware/02.hero/boards/uno-r4-minima/tutorials/usb-hid/usb-hid.md:18]: This should be "are" since it is multiple learning goals

Comment (jcarolinares) [reply content/hardware/02.hero/boards/uno-r4-minima/tutorials/usb-hid/usb-hid.md:18]: we are mixing yes let me fix it

Comment (jcarolinares) [reply content/hardware/02.hero/boards/uno-r4-minima/tutorials/usb-hid/usb-hid.md:18]: you are right

Comment (jcarolinares) [inline content/hardware/02.hero/boards/uno-r4-minima/tutorials/usb-hid/usb-hid.md:18]:
```suggestion
The goals of this tutorial is:
```

Comment (jacobhylen) [inline content/hardware/02.hero/boards/uno-r4-minima/tutorials/usb-hid/usb-hid.md:18]:
```suggestion
The goals of this tutorial are to:
```

**Review by jacobhylen — APPROVED** [review]

**Pull request merged**